crates/aptos/src/node/mod.rs

```diff
@@ -188,7 +188,7 @@ impl CliCommand<Transaction> for IncreaseLockup {
         self.txn_options
             .submit_script_function(
                 AccountAddress::ONE,
-                "Stake",
+                "stake",
```

Comment:
This is a duplicate of https://github.com/aptos-labs/aptos-core/pull/2234 which is more comprehensive.